coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Language and docstyle independent documentation comments extraction.
+
+Each of the functions is built upon one another, and at the last,
+exposes a single function py:function:`extract_documentation_with_markers`
```

Comment:
@Asalle you didn't add this reference https://stackoverflow.com/a/7754150/3578554